R-package/README.md

```diff
@@ -33,7 +33,7 @@ Install LightGBM R-package with the following command:
 ```sh
 git clone --recursive https://github.com/Microsoft/LightGBM
 cd LightGBM/R-package
-# export CXX=g++-7 CC=gcc-7 # for OSX
+#  export CC=/usr/local/bin/gcc CXX=/usr/local/bin/g++ CXX1X=/usr/local/bin/g++ # for OSX
```

Comment:
if you install gcc via Homebrew, `/usr/local/bin/gcc` and `/usr/local/bin/g++` don't exist.

```
~
❯ ls /usr/local/bin/gcc
ls: /usr/local/bin/gcc: No such file or directory

~
❯ ls /usr/local/bin/gcc-*
/usr/local/bin/gcc-6@        /usr/local/bin/gcc-ar-6@     /usr/local/bin/gcc-nm-6@     /usr/local/bin/gcc-ranlib-6@
/usr/local/bin/gcc-7@        /usr/local/bin/gcc-ar-7@     /usr/local/bin/gcc-nm-7@     /usr/local/bin/gcc-ranlib-7@
```